neural_style.py

```diff
@@ -10,15 +10,25 @@
 import math
 from argparse import ArgumentParser
 
+from matplotlib import pyplot as plt
```

Comment:
Fixed in https://github.com/anishathalye/neural-style/pull/50/commits/393117bd336947d1dcf0a4061657f57000d4bc54
PIL image quality setting works well, no warnings when save to other formats too